nvflare/fuel/f3/communicator.py

```diff
@@ -30,24 +31,31 @@
 
 log = logging.getLogger(__name__)
 _running_instances = weakref.WeakSet()
+driver_mgr = DriverManager()
+driver_loaded = False
 
 
-def _exit_func():
-    for c in _running_instances:
-        c.stop()
-        log.debug(f"Communicator {c.local_endpoint.name} was left running, stopped on exit")
+def load_comm_drivers():
+    global driver_loaded
```

Comment:
With this global "driver_loaded" variable, we can only scan and load all the drivers once within the system set up. Do we want to enhance this to individual driver control? In this case, we will be able to support the use case of potential individual job uses different communication driver. The job specific driver will be placed in the job "custom" folder. And the additional driver will be dynamically loaded when the job is to run. The DriverManager can hold a dict to show the drivers have been loaded.